scripts/pre_commit_linter.py

```diff
@@ -2284,6 +2285,49 @@ def _check_codeowner_file(self):
 
         return summary_messages
 
+    def _check_extra_js_files(self):
+        """Checks if the changes made include extra js files in core
+        or extensions folder which are not specified in
+        build.JS_FILEPATHS_NOT_TO_BUILD.
+        """
+
+        if self.verbose_mode_enabled:
+            print 'Starting extra js files check'
+            print '----------------------------------------'
+
+        summary_messages = []
+        failed = False
+        with _redirect_stdout(_TARGET_STDOUT):
+            js_files_to_check = [
+                filepath for filepath in self.all_filepaths if (
+                    filepath.endswith('.js'))]
+
+            for filename in js_files_to_check:
```

Comment:
filename --> filepath (right)?

(Filename is just the last part, I think.)

Reply:
Yes, done.